README.md

```diff
@@ -29,14 +29,14 @@ Once unzipped, right-click the font file and click "Install for all users". This
 For more details and app-specific instructions, [please check the wiki](https://github.com/microsoft/cascadia-code/wiki/Installing-Cascadia-Code). 
 
 # Get involved
-Instructions on how to modify and submit an update to the Cascadia Code source is [available in the wiki](https://github.com/microsoft/cascadia-code/wiki/Modifying-Cascadia-Code).
+Instructions on how to modify and submit an update to the Cascadia Code source are [available in the wiki](https://github.com/microsoft/cascadia-code/wiki/Modifying-Cascadia-Code).
 
 
 # Communicating with the Team
 
-The easiest way to communicate with the team is via GitHub issues. Please file new issues, feature requests and suggestions, but **DO search for similar open/closed pre-existing issues before you do**.
+The easiest way to communicate with the team is via GitHub Issues. Please file new issues, feature requests, and suggestions, but **DO search for similar open/closed pre-existing issues before you do**.
 
-Please help us keep this repository clean, inclusive, and fun! We will not tolerate any abusive, rude, disrespectful or inappropriate behavior. Read our [Code of Conduct](https://opensource.microsoft.com/codeofconduct/) for more details.
+Please help us keep this repository clean, inclusive, and fun! We will not tolerate any abusive, rude, disrespectful, or inappropriate behavior. Read our [Code of Conduct](https://opensource.microsoft.com/codeofconduct/) for more details.

```

Comment:
The Oxford comma is a matter of personal taste. :)

Reply:
@DHowett This repo is not eligible for hactoberfest.:)